### PR TITLE
Docs: Update note on Next.js ignoring CRA files

### PR DESCRIPTION
**Pull request by delbaoliveira**
Adding clarity relating to: https://github.com/vercel/next.js/pull/66959#issuecomment-2180925616

### CODE REVIEW

**Review by samcx — COMMENTED** [review]
Thanks for the edit!

The part that's still confusing → "additional iconography other than the favicon"—is this related to all files that are not the ones listed here → https://nextjs.org/docs/app/building-your-application/optimizing/metadata#file-based-metadata? If yes, maybe we can add a link to this, and stating these are the only ones we check.

File: docs/02-app/01-building-your-application/11-upgrading/05-from-create-react-app.mdx [hunk 1]
```diff
@@ -143,7 +143,7 @@ export default function RootLayout({ children }) {
 }
 ```
 
-> **Good to know**: We'll ignore the [manifest file](/docs/app/api-reference/file-conventions/metadata), additional iconography other than the favicon, and [testing configuration](/docs/app/building-your-application/testing), but if these are requirements, Next.js also supports these options.
+> **Good to know**: Next.js ignores CRA's `public/manifest.json` file, additional iconography other than the favicon, and [testing configuration](/docs/app/building-your-application/testing), but if these are requirements, Next.js also supports these options. See the [Metadata API](/docs/app/building-your-application/optimizing/metadata) and [Testing](/docs/app/building-your-application/testing) docs for more information.
```
Comment (delbaoliveira) [inline docs/02-app/01-building-your-application/11-upgrading/05-from-create-react-app.mdx:146]:
```suggestion
> **Good to know**: Next.js ignores CRA's `public/manifest.json` file, additional iconography other than the [favicon](/docs/app/api-reference/file-conventions/metadata/app-icons) and [testing configuration](/docs/app/building-your-application/testing), but if these are requirements, Next.js also supports these options. See the [Metadata API](/docs/app/building-your-application/optimizing/metadata) and [Testing](/docs/app/building-your-application/testing) docs for more information.
```

**Comment by delbaoliveira** [comment]
Yes, it's related to the [favicon.ico](https://nextjs.org/docs/app/api-reference/file-conventions/metadata/app-icons) file. Added a link.

**Review by samcx — COMMENTED** [review]
> additional iconography other than the [favicon]

Isn't this implying that we ignoring almost everything besides the `favicon`, when it's really we ignore everything except the favicon, **icon**, and **apple-icon** (bolded ones we should also mention)?

File: docs/02-app/01-building-your-application/11-upgrading/05-from-create-react-app.mdx [hunk 2]
```diff
@@ -143,7 +143,7 @@ export default function RootLayout({ children }) {
 }
 ```
 
-> **Good to know**: We'll ignore the [manifest file](/docs/app/api-reference/file-conventions/metadata), additional iconography other than the favicon, and [testing configuration](/docs/app/building-your-application/testing), but if these are requirements, Next.js also supports these options.
+> **Good to know**: Next.js ignores CRA's `public/manifest.json` file, additional iconography other than the [favicon](/docs/app/api-reference/file-conventions/metadata/app-icons) and [testing configuration](/docs/app/building-your-application/testing), but if these are requirements, Next.js also supports these options. See the [Metadata API](/docs/app/building-your-application/optimizing/metadata) and [Testing](/docs/app/building-your-application/testing) docs for more information.
```
Comment (delbaoliveira) [inline docs/02-app/01-building-your-application/11-upgrading/05-from-create-react-app.mdx:146]:
```suggestion
> **Good to know**: Next.js ignores CRA's `public/manifest.json` file, additional iconography (except [favicon](/docs/app/api-reference/file-conventions/metadata/app-icons)), and [testing configuration](/docs/app/building-your-application/testing), but if these are requirements, Next.js also supports these options. See the [Metadata API](/docs/app/building-your-application/optimizing/metadata) and [Testing](/docs/app/building-your-application/testing) docs for more information.
```

**Review by samcx — APPROVED** [review]
<picture data-single-emoji=":lfgtm:" title=":lfgtm:">:lfgtm:</picture>!

**Pull request merged**